### PR TITLE
Screenshots and the descriptions of style tab

### PR DESCRIPTION
**Pull request by prog8**
Chart Builder

### CODE REVIEW

**Comment by otisg** [comment]
What I miss here is how to get started with CB.  Where in the UI do I go to access this CB?  Can we show that?

**Comment by prog8** [comment]
Sure, we show that.

**Comment by prog8** [comment]
@otisg 

chart-builder-2

**Comment by otisg** [comment]
Ah, OK.  Didn't see that.

**Comment by prog8** [comment]
You didn't because this is a new thing I created after your comment. I'd like you to see this and comment if this looks OK.

**Comment by otisg** [comment]
Aha.  I think an MP4 would be a little better.  That "arrow" looks very unusual. :)

**Comment by prog8** [comment]
That arrow is taken from the other screenshot we have in docs but I can create mp4 too

**Comment by prog8** [comment]
Chart Builder (1)

File: docs/dashboards/chart-builder.md [hunk 1]
```diff
@@ -176,3 +190,18 @@ Variables can be created in Report options "Manage variables".
 
 ![Manage Variables](../images/dashboards/variables.png)
 
+
+## Style
+
+In the style tab, you configure what the chart looks like. You can select bar, line, area and point 
+charts. In addition to chart type, you can choose a color from the palette and group color scheme. The main
```
Comment (adnanrahic) [inline docs/dashboards/chart-builder.md]: charts. In addition to chart type, you can choose a color from the palette and group color scheme. The main
=>
charts. In addition to the chart type, you can choose a color from the palette and group color scheme. The main

File: docs/dashboards/chart-builder.md [hunk 2]
```diff
@@ -176,3 +190,18 @@ Variables can be created in Report options "Manage variables".
...
+goal of a group color scheme is to determine the colors of the series from the same group. When you group metric by
```
Comment (adnanrahic) [inline docs/dashboards/chart-builder.md]: goal of a group color scheme is to determine the colors of the series from the same group. When you group metric by
=>
goal of a group color scheme is to determine the colors of the series from the same group. When you group metrics by

File: docs/dashboards/chart-builder.md [hunk 3]
```diff
@@ -176,3 +190,18 @@ Variables can be created in Report options "Manage variables".
...
+`os.host` as a result you get a couple of series. Each series can get a different color.
+
+![Chart builder styles](../images/dashboards/chart-builder-styles.png)
+
+A very important feature in the styles tab is "Stacked" option. When you select the checkbox data
```
Comment (adnanrahic) [inline docs/dashboards/chart-builder.md]: A very important feature in the styles tab is "Stacked" option. When you select the checkbox data
=>
A very important feature in the styles tab is the "Stacked" option. When you select the checkbox data

File: docs/dashboards/chart-builder.md (hunk 2 above)
Comment (prog8) [reply docs/dashboards/chart-builder.md]: I intentionally used `metric` because you group a single metric by a tag

Comment (adnanrahic) [reply docs/dashboards/chart-builder.md]: Aha! Then it should be "a metric". Add an "a" before `metric` and you're good to go. Merge away. :smile:

**Pull request merged**